CMakeLists.txt

```diff
@@ -78,7 +79,6 @@ else()
   add_cxx_compiler_flag(-pedantic-errors)
   add_cxx_compiler_flag(-Wshorten-64-to-32)
   add_cxx_compiler_flag(-Wfloat-equal)
-  add_cxx_compiler_flag(-Wzero-as-null-pointer-constant)
```

Comment:
I'll just disable it when `BENCHMARK_USE_LIBCXX` is `ON`.

As I side note I dislike this warning in general. The warning seems to take into account constant propagation when it should really only warn on the literal token '0'.